guides/common/modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc

```diff
@@ -0,0 +1,21 @@
+[id="configuring-selinux-to-permit-access-to-repositories-on-custom-ports_{context}"]
+= Configuring SELinux to Permit Access to Repositories on Custom Ports
+
+SELinux permits access of {Project} to repositories only on specific ports.
```

Comment:
```suggestion
SELinux permits access of {Project} for content syncs only on specific ports.
```

Reply:
reworded to avoid using term "repositories"